bert_e/git_host/github/__init__.py

```diff
@@ -624,7 +634,7 @@ def remove_unwanted_workflows(self):
         self._check_suites = list(map(lambda elem: {
             **elem,
             'html_url':
-                dispatched.get(elem['id'], {'html_url': ''})['html_url']
+                dispatched.get(elem['id'], '')['html_url']
```

Comment:
Keep the old version of this line. 
WIth a astring as default parameter `''['html_url']` will raise an exception, the original version won't.